dual_net.py

```diff
@@ -407,28 +402,32 @@ def my_res_layer(inputs):
     return policy_output, value_output, logits
 
 
-def get_estimator(working_dir):
+def get_estimator():
     if FLAGS.use_tpu:
-        return get_tpu_estimator(working_dir)
+        return _get_tpu_estimator()
+    else:
+        return _get_nontpu_estimator()
+
 
+def _get_nontpu_estimator():
     run_config = tf.estimator.RunConfig(
         save_summary_steps=FLAGS.summary_steps,
         keep_checkpoint_max=FLAGS.keep_checkpoint_max)
     return tf.estimator.Estimator(
         model_fn,
-        model_dir=working_dir,
+        model_dir=FLAGS.model_dir,
```

Comment:
Does this mean that non-tpu training will spew `model.ckpt-123456` into gs://$BUCKET/models/ ....?

Reply:
Because (here and below) conflating the fsdb 'model_dir' with the 'work_dir' is probably not the right thing...